static/js/containers/pages/login/LoginPasswordPage.js

```diff
@@ -50,11 +50,11 @@ class LoginPasswordPage extends React.Component<Props> {
 
     try {
       const {
-        body: { state, redirectUrl, errors }
+        body: { state, redirect_url, errors }
       }: { body: AuthResponse } = await loginPassword(password, partialToken)
```

Comment:
I think this needs to be typed as `AuthRawResponse` to fix the flow/lint errors.

Reply:
fixed build.